test/Microsoft.ML.TestFramework/Attributes/MatrixFactorizationFactAttribute.cs

```diff
@@ -17,7 +17,7 @@ public MatrixFactorizationFactAttribute() : base("Disabled - this test is being
         /// <inheritdoc />
         protected override bool IsEnvironmentSupported()
         {
-            return Environment.Is64BitProcess;
+            return true;
```

Comment:
Can probably just remove this whole thing, I think.

Reply:
I'd keep it. We will be enabling LIBMF's SSE/AVX and I guess this might be useful if LIBMF's numerical stability changes.

Reply:
Can you at least remove `base("Disabled - this test is being fixed as part of https://github.com/dotnet/machinelearning/issues/1441")`, as that appears to no longer be valid?

Reply:
No problem.